docs/toolbox/github-advanced-security/index.md

```diff
@@ -18,3 +18,47 @@ Check out [this repository](https://github.com/equinor/appsec-ghas-examples) for
 **Container Scanning**: Please note that GHAS does not currently offer container scanning solutions, we have explored other alternatives like Trivy so please reach out if you have any questions.
 
 **GHAS Features**: We encourage you to enable additional GHAS features like code scanning with CodeQL 
+
+!!! info
+    The following information offers guidance to internal Equinor teams and might not apply to the broader public.
+
+## Compliance with Governance
+
+To comply with [TR2375](https://docmap.equinor.com/Docmap/page/doc/dmDocIndex.html?DOCKEYID=1000005127) you must perform:
+
+- **SCA** (Software Composition Analysis) - dependency scanning, handled by Dependabot.
+- **SAST** (Static Application Security Testing) - source code scanning, handled by Code Scanning (CodeQL).
+
+In the Equinor GitHub Organization:
+
+- **Secret Scanning** is enabled by default for all repositories 
+- **Dependency Scanning** is enabled by default for most repositories
+- **Code Scanning** must be set up manually by a repository Admin
+
+### What do you need to do?
+
+1. Check dependency scanning (SCA)
+    In your repository go to Security -> Dependency Scanning and make sure it's active (alerts created)
+1. Enable CodeQL (SAST)
+    CodeQL is not enabled by default. A repository Admin must:
+    - Navigate to Settings → Advanced Security → CodeQL analysis → Set up → Default
+    - If the default setup fails, you will see a message under Security → Code Scanning.
+      In that case, use an advanced setup. Examples of this are available in the [appsec-ghas-examples repository](https://github.com/equinor/appsec-ghas-examples).
+
+      ![Code analysis tools not working](image-3.png)
+
+!!! warning
+
+    If you ignore your Dependabot alerts for too long, **Dependabot will stop working**.
+    In this case, you will see the following warning under Security → Dependabot.
+    This means you will be **noncompliant** with SCA requirements.
```

Comment:
I do not think this is entirely true 🤔 
Dependabot is _not_ SCA.

Dependabot is for updating dependencies, not getting security alerts or having the dependency graph (the SCA-part)....

Reply:
The dependency graph is a "snap shot" of the _current state_ of the dependencies. So the alerting would still go on, I am pretty sure :)